### PR TITLE
elixir_client: create package

### PR DESCRIPTION
**Pull request by predrag-rakic**

### CODE REVIEW

File: lib/thrifter/git_repo.ex [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    git_cmd(~w(pull origin master))
+  end
+
+  defp git_cmd(args) do run("git", args) end
+
+  defp run(cmd, args) do
+     run(cmd, args, @debug)
+  end
+
+  defp run(cmd, args, true) do
```
Comment (shiroyasha) [inline lib/thrifter/git_repo.ex]: Maybe use `defp run(cmd, args, debug: true) do`. Without that it is hard to figure out what is `true`.

File: lib/thrifter/git_repo.ex [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          "Creating package #{package}\n",
+          f)
+
+    Mix.shell.info("Package #{package} created in #{Colors.green(codecommit)}\n")
+  end
+
+  defp pushd(dir, message, f) do
+    Mix.shell.info(message)
+
+    {:ok, old_wd} = File.cwd
```
Comment (shiroyasha) [inline lib/thrifter/git_repo.ex:31]: what is `wd`?

File: lib/thrifter/git_repo.ex [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    end
+
+    pushd(client_dir,
+          "Creating package #{package}\n",
+          f)
+
+    Mix.shell.info("Package #{package} created in #{Colors.green(codecommit)}\n")
+  end
+
+  defp pushd(dir, message, f) do
```
Comment (shiroyasha) [inline lib/thrifter/git_repo.ex:28]: What is `f`, `format` ?

**Comment by shiroyasha** [comment]
Looks nice 👍

File: lib/thrifter/git_repo.ex (hunk 1 above)
Comment (predrag-rakic) [reply lib/thrifter/git_repo.ex]: :+1:

**Pull request merged**